app/controllers/crops_controller.rb

```diff
@@ -215,6 +215,15 @@ def destroy
   private
 
   def crop_params
-    params.require(:crop).permit(:en_wikipedia_url, :name, :parent_id, :creator_id, :approval_status, :request_notes, :reason_for_rejection, :rejection_notes, scientific_names_attributes: [:scientific_name, :_destroy, :id])
+    params.require(:crop).permit(:en_wikipedia_url,
+                                 :name,
+                                 :parent_id,
+                                 :creator_id,
+                                 :approval_status,
+                                 :request_notes,
+                                 :reason_for_rejection,
+                                 :rejection_notes, scientific_names_attributes: [:scientific_name,
```

Comment:
Regardless of what we decide about indentation style, I think you probably want a newline before `scientific_names_attributes` here. Depending on the outcome of the indentation style debate, you may or may not need one after `[`.